api/v1/controllers/TrackingController.js

```diff
@@ -0,0 +1,44 @@
+var bodyParser = require('body-parser');
+
+var services = require('../services');
+var middleware = require('../middleware');
+var requests = require('../requests');
+var roles = require('../utils/roles');
+
+var router = require('express').Router();
+
+function createTrackedEvent(req, res, next) {
+    services.TrackingService
+        .createTrackingEvent(req.body)
+        .then(function (result) {
+            res.body = result.toJSON();
+
+            return next();
+        })
+        .catch(function (error) {
+            return next(error);
+        });
+}
+
+function addTrackedEventParticipant(req, res, next) {
```

Comment:
It lets a volunteer figure out if an attendee has already partaken in the current tracked event, it returns a 200 response if they haven't, if they have it throws an error.